ios/Classes/MapboxMapController.swift

```diff
@@ -606,11 +661,15 @@ class MapboxMapController: NSObject, FlutterPlatformView, MGLMapViewDelegate, Ma
             guard let properties = arguments["properties"] as? [String: String] else { return }
             let belowLayerId = arguments["belowLayerId"] as? String
             let sourceLayer = arguments["sourceLayer"] as? String
+            let minzoom = arguments["minzoom"] as? Double
+            let maxzoom = arguments["maxzoom"] as? Double
             addFillLayer(
                 sourceId: sourceId,
                 layerId: layerId,
                 belowLayerId: belowLayerId,
                 sourceLayerIdentifier: sourceLayer,
+                minimumZoomLevel: minzoom != nil ? Float(minzoom!) : nil,
```

Comment:
any reason for casting this again? minzoom is a Double? at this point

[minimumZoomLevel](https://docs.mapbox.com/ios/maps/api/6.3.0/Classes/MGLMapView.html#/c:objc(cs)MGLMapView(py)minimumZoomLevel) is double as well

Reply:
It is because [minimumZoomLevel on MGStyleLayer](https://docs.mapbox.com/ios/maps/api/6.3.0/Classes/MGLStyleLayer.html#/c:objc(cs)MGLStyleLayer(py)minimumZoomLevel) is a different type than [minimumZoomLevel on MGLMapView](https://docs.mapbox.com/ios/maps/api/6.3.0/Classes/MGLMapView.html#/c:objc(cs)MGLMapView(py)minimumZoomLevel) for some reason.

Reply:
might still be more elegant to pass it as a double and only cast it where required.

Reply:
Done